jol-cli/src/test/java/org/openjdk/jol/operations/ClasspathedOperationTest.java

```diff
@@ -0,0 +1,82 @@
+package org.openjdk.jol.operations;
+
+import org.junit.Test;
+
+import java.util.Objects;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+/** Tests for {@link ClasspathedOperation}. */
+public final class ClasspathedOperationTest {
+
+    @Test
+    public void testTryInstantiate() throws Exception {
+
+        final ClasspathedOperation classpathedOperation = new ClasspathedOperation() {
```

Comment:
```suggestion
        final ClasspathedOperation op = new ClasspathedOperation() {
```

...and the rest of the renames.